tools/checkswitch/checkswitch.go

```diff
@@ -146,6 +147,69 @@ func run(pass *analysis.Pass) (any, error) {
 						}
 					}
 				}
+			// Handle *ast.SwitchStmt, which represents a switch statement in Go.
+			case *ast.SwitchStmt:
+				// Declare a variable to hold the name of the case.
+				var name string
+
+				// Iterate over each case in the switch statement.
+				for _, el := range n.Body.List {
+					// If a case has no expressions (i.e., it's a default case), skip to the next case.
```

Comment:
Do we actually have `default` cases? I think we have linter enabled to check that `switch`es are exhaustive. Maybe that tool should double-check that too.